tests/common/plugins/test_completeness/README.md

```diff
@@ -0,0 +1,68 @@
+### CompletenessLevel markers
+CompletenessLevel marker enables testcases to be executed in different meaningful levels.
+Each level is a representation of the scope of execution of a testcase. This document describes the usage of CompletenessLevel marker.
+
+### To use CompletenessLevel:
+- Use pytest command line option ```--test_completeness_level```
+- Identified meaningful levels - 
+        
+        0. Debug
+        
+        1. Basic
+        
+        2. Confident
+        
+        3. Thorough
+- Mark the testcase with marker ```supported_completeness_level```
+
+### Different cases for CompletenessLevel
+
+    1. Completeness level not specified - run the lowest defined level (full test if no defined level)
+
+    2. Test does not define any completeness level - run the full testcase
+    
+    3. Specified completeness level do not match any defined level in a test case:
+        3.1 Specified level is higher than any defined level - go to highest level defined
+        3.2 Specified level is lower than any defined level - go to lowest level defined
+        3.3 Specified level is in between two defined levels - go to next lower level
+    
+    4. Specified level matches one of the defined levels
+
+### CompletenessLevel usage example
+```python
+import pytest
+from tests.common.plugins.test_completeness import CompletenessLevel
+
+pytestmark = [pytest.mark.supported_completeness_level(CompletenessLevel.Debug, CompletenessLevel.Thorough)]
```

Comment:
I don't think using marker to implement the completeness level is the right approach.

Reply:
This concern is addressed in other comment  here -https://github.com/Azure/sonic-mgmt/pull/1927#discussion_r458420334